mods/ts/rules/aircraft.yaml

```diff
@@ -90,7 +92,6 @@ ORCA:
 		TakeoffSounds: orcaup1.aud
 		LandingSounds: orcadwn1.aud
 		AltitudeVelocity: 128
-		FlightDynamics: VTOL, Hover, TakeOffOnCreation, TakeOffOnResupply, TurnToDock
```

Comment:
I thought it was intentional to not be able to slide here? If so, `CanSlide: false` should be added (or the default changed).